test/API/test_calls.py

```diff
@@ -0,0 +1,222 @@
+import os
+import sys
+sys.path.insert(0, os.path.abspath( os.path.join(os.path.dirname(__file__), '../../src/') ))
+
+from Zoomba.APILibrary import APILibrary
+import unittest
+from unittest.mock import patch
+from unittest.mock import PropertyMock
+
+
+class TestInternal(unittest.TestCase):
+    def test_suppress_default(self):
+        library = APILibrary()
+        assert not library.suppress_warnings
+
+    def test_suppress_set(self):
+        library = APILibrary()
+        library.suppress_insecure_request_warnings("True")
+        assert library.suppress_warnings
+
+    def test_suppress_set_default(self):
+        library = APILibrary()
+        library.suppress_insecure_request_warnings()
+        assert library.suppress_warnings
+
+    def test_suppress_set_false(self):
+        library = APILibrary()
+        library.suppress_warnings = True
+        library.suppress_insecure_request_warnings("False")
+        assert not library.suppress_warnings
+
+
+class TestExternal(unittest.TestCase):
+    def test_get_default(self):
+        library = APILibrary()
+        self.assertRaises(TypeError, library.call_get_request)
+
+    @patch('RequestsLibrary.RequestsKeywords.create_session')
+    @patch('RequestsLibrary.RequestsKeywords.get_request')
+    def test_basic_get(self, create_session, get_request):
+        library = APILibrary()
+        r = library.call_get_request({"a": "Text"}, "Endpoint", "fullstring")
+        type(r).text = PropertyMock(return_value="success")
+        type(r).status_code = PropertyMock(return_value=200)
+        assert r.text
```

Comment:
Shouldn't this be `assert r.text == "success"`

If so - multiple occurrences throughout file.

Reply:
yea, I totally forgot about this one